### PR TITLE
Login, Registration and some css

### PR DESCRIPTION
**Pull request by REV3NSS**
Added Login, Registration and some css.

### CODE REVIEW

**Review by rs-piotr-kryska — CHANGES_REQUESTED** [review]
Please do no push pyc files (and db.sqlite)

File: league/manager/forms.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	class Meta:
+		model = User
+		fields = ("username", "email", "password1", "password2")
+
+	def save(self, commit=True):
+		user = super(NewUserForm, self).save(commit=False)
+		user.email = self.cleaned_data['email']
+		if commit:
+			user.save()
+		return user
```
Comment (rs-piotr-kryska) [inline league/manager/forms.py:20]: Its strangethat it is necessary

File: league/manager/migrations/0003_alter_league_max_number_of_teams.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        ('manager', '0002_eventtype_result_sumofpoints_team_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='league',
+            name='max_number_of_teams',
+            field=models.IntegerField(),
+        ),
+    ]
```
Comment (rs-piotr-kryska) [inline league/manager/migrations/0003_alter_league_max_number_of_teams.py:18]: Please remove migrations

**Comment by Shefour** [comment]
@skam34 can you look at this

**Pull request closed without merging**